packages/edit-post/src/components/sidebar/post-link/index.js

```diff
@@ -53,38 +53,46 @@ function PostLink( {
 			onToggle={ onTogglePanel }
 		>
 			{ isEditable && (
-				<TextControl
-					label={ __( 'URL' ) }
-					value={ forceEmptyField ? '' : currentSlug }
-					onChange={ ( newValue ) => {
-						editPermalink( newValue );
-						// When we delete the field the permalink gets
-						// reverted to the original value.
-						// The forceEmptyField logic allows the user to have
-						// the field temporarily empty while typing.
-						if ( ! newValue ) {
-							if ( ! forceEmptyField ) {
+				<div className="editor-post-link">
```

Comment:
It looks like the className `editor-post-link` doesn't have any associated styles. It can probably be removed unless there's a good reason for it?

Reply:
Is this div really necessary or should we just replace it with `<Fragment />`